place_tracker/README.md

```diff
@@ -1,6 +1,30 @@
 # Place Tracker
 
-A sample place tracking app that uses the [google_maps_flutter](https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter) plugin.
+A sample place tracking app that uses the [google_maps_flutter](https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter) plugin. 
+Keep track of your favorite places, places you've visited, and places you want to go. View details
+about these places, show them on a map, and get directions to them.
+
+**This sample is not currently in a finished state. We're in the process
+of building it out. This sample currently only works on Android (see Caveat below).**
+
+## Goals
+
+* Learn how to show, control, and modify a GoogleMap widget.
+* Learn how to place a marker on a map.
+* Learn how to incorporate the GoogleMap widget with other Flutter widgets.
```

Comment:
nit: A more "flutter" way of putting this might be "Learn how to create an interface composed of GoogleMap and other widgets."

Reply:
Modified phrasing. Will merge once tests pass.